src/Service/Gateway.php

```diff
@@ -92,10 +93,22 @@ public function pay(
             $pre_order = $this->processOrder($transaction,$salesChannelContext);
             $order = $this->ginger->createOrder($pre_order);
 
+            /**
+             * Condition if order created with error status
+             */
             if($order['status'] == 'error') {
                 $this->transactionStateHandler->fail($transaction->getOrderTransaction()->getId(), $salesChannelContext->getContext());
                 throw new EmsPluginException(current($order['transactions'])['reason']);
             }
+
+            /**
+             * Redirect for bank-transfer payment method
+             */
+            if (isset($order['transactions']) && current($order['transactions'])['payment_method'] == 'bank-transfer' && current($order['transactions'])['status']){
+                return new RedirectResponse(
+                implode('&',[$order['return_url'],implode('=',['order_id',$order['id']]),implode('=',['project_id',$order['project_id']])])
```

Comment:
The code is difficult to read. Why not use the usual cancotation of several parts without using the implode function?

Reply:
Will don